static/sass/_pattern_careers-modal.scss

```diff
@@ -0,0 +1,35 @@
+@mixin canonical-p-careers-modal {
+  #contact-modal {
+    // sass-lint:disable-line no-ids
+    &.p-modal {
+      align-items: start;
+      position: fixed;
+      z-index: 100;
```

Comment:
I think the modal is better appearing in the centre of the screen. I don't think you need these styles.